### ISSUE TITLE
 Please tell me How to make lua files open lua?

### ISSUE THREAD

**Issue by egor230** [aviatesk/atom-open-in-app#21]
Thank you very much for your package.  It helps solve my problem. Please tell me How to make lua files open lua?

 Where should I write?```

Thank you very much for your package.  It helps solve my problem. Please tell me How to make lua files open lua?

 Where should I write?.lua": "C:/Program Files/lua.exe"
```

**Comment by aviatesk** [comment]
Described in https://github.com/aviatesk/atom-open-in-app#configuration-example

e.g.

> your config file
```coffee
"*":
  ...
  "open-in-app":
    applicationBindings: { ".lua": "C:\\Program FIles\\lua.exe" }
  ...
```

**Comment by egor230** [comment]
thank you very much for your answer,I'm just starting to master the atom editor, so I'm still an inexperienced user, I wrote these values, maybe I did something wrong, I'll stick a screenshot.please help me figure this out.I really need your advice
![Screenshot](https://user-images.githubusercontent.com/32921166/73463512-30742300-439f-11ea-8891-a79c1ab9c5e9.png)

**Comment by aviatesk** [comment]
No, open [your config file](https://flight-manual.atom.io/using-atom/sections/basic-customization/#global-configuration-settings), and put the above snippet into that.

**Comment by egor230** [comment]
thank you very much for your patience and help. tried both, not working out, like doing everything according to instructions. I recorded a video as I try to configure this package. https://youtu.be/2h0v4yLB-Xw

**Comment by aviatesk** [comment]
In CoffeeScript (the `.cson` files), indent has important meanings, so please just use 2-space indent for this configuration if you use that for the other settings.
And also, according to the video, you seem to try to open `.cson` files, but `applicationBindings: { ".lua": "C:\\Program FIles\\lua.exe" }` means opening `.lua` files in `lua.exe`

**Comment by egor230** [comment]
thank you very much now for your response. I'm sorry, I'm bothering you again, with my stupid one. I deleted atom and installed the latest version, downloaded the package, wanted to do everything correctly. at least I tried. got the following error.   'cmd' could not be spawned. Is it installed and on your path? If so please open an issue on the package spawning the process. I wanted to know why it occurs and how to fix it. please help me, I ask you very much https://www.youtube.com/watch?v=Unfe8i0DlqY&feature=youtu.be
```

activate aligner-lua
events.js:167 Uncaught Error: spawn cmd ENOENT
    at Process.ChildProcess._handle.onexit (internal/child_process.js:232)
    at onErrorNT (internal/child_process.js:407)
    at process._tickCallback (internal/process/next_tick.js:63)
```
